src/Components/about/AboutUs.jsx

```diff
@@ -106,90 +123,83 @@ export default function AboutUs() {
-							color="textSecondary"
-							component="p"
-							gutterBottom
-						>
-							Fill up the form and our team will get back to you...
-						</Typography>
-						<form>
-							<Grid container spacing={1} justify="center" alignItems="center">
-								<Grid xs={12} sm={6} item>
-									<TextField
-										placeholder="Enter first name"
-										label="First Name"
-										variant="outlined"
-										fullWidth
-										required
-									/>
-								</Grid>
-								<Grid xs={12} sm={6} item>
-									<TextField
-										placeholder="Enter last name"
-										label="Last Name"
-										variant="outlined"
-										fullWidth
-										required
-									/>
-								</Grid>
-								<Grid item xs={12}>
-									<TextField
-										type="email"
-										placeholder="Enter email"
-										label="Email"
-										variant="outlined"
-										fullWidth
-										required
-									/>
-								</Grid>
-								<Grid item xs={12}>
-									<TextField
-										type="number"
-										placeholder="Enter phone number"
-										label="Phone"
-										variant="outlined"
-										fullWidth
-										required
-									/>
-								</Grid>
-								<Grid item xs={12}>
-									<TextField
-										label="Message"
-										multiline
-										rows={4}
-										placeholder="Type your message here"
-										variant="outlined"
-										fullWidth
-										required
-									/>
-								</Grid>
-								<Grid item xs={12}>
-									<Button
-										type="submit"
-										variant="contained"
-										color="primary"
-										fullWidth
-									>
-										Submit
-									</Button>
-								</Grid>
+				<Paper className={classes.paper} elevation={4}>
+					<Typography gutterBottom variant="h5" align="center">
+						Contact Us
+					</Typography>
+					<Typography
+						variant="body2"
+						color="textSecondary"
+						component="p"
+						align="center"
+						gutterBottom
+					>
+						Fill up the form and our team will get back to you...
+					</Typography>
+					<form>
+						<Grid container spacing={1} className={classes.align}>
+							<Grid item xs={12} sm={6}>
+								<TextField
+									fullWidth
+									placeholder="Enter first name"
+									label="First Name"
+									variant="outlined"
+									required
+								/>
+							</Grid>
+							<Grid item xs={12} sm={6}>
+								<TextField
+									fullWidth
+									placeholder="Enter last name"
+									label="Last Name"
+									variant="outlined"
+									required
+								/>
+							</Grid>
+							<Grid item xs={12}>
+								<TextField
+									fullWidth
+									type="email"
+									placeholder="Enter email"
+									label="Email"
+									variant="outlined"
+									required
+								/>
+							</Grid>
+							<Grid item xs={12}>
+								<TextField
+									fullWidth
+									type="number"
+									placeholder="Enter phone number"
+									label="Phone"
+									variant="outlined"
+									required
```

Comment:
using type=" number" can create problems here like you can type letters "e" and "E" even if you have defined it as the number and you can even add + - characters so using it makes no sense.

Reply:
Agreed. 
Shall i implement some kind of custom form validator or is there an inbuilt type that lets us handle this prob..
what do u suggest?

Reply:
I think you should do custom form validation, as it will help you to do it in a more creative way.

Reply:
@moinak878 In contact us the phone number is not necessary you can remove it.